geopandas/geoseries.py

```diff
@@ -767,14 +760,21 @@ def plot(self, *args, **kwargs):
 
     plot.__doc__ = plot_series.__doc__
 
-    def explode(self):
+    def explode(self, add_multiindex=True):
```

Comment:
Can we have `ignore_index` in here as well? We want these two to be consistent.

Reply:
Done